package.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
All configs, documentation, gitignore changes should not be in Login commit. Try next.

1. git reset --soft HEAD~1
2. via GUI interface of your IDEA chose configs, documentation, gitignore files. Create commit for it.
3. then chose all files related to Login and commit it
4. git push -f